chainstate/test-framework/src/utils.rs

```diff
@@ -264,3 +264,27 @@ fn new_token_burn_output(
         OutputPurpose::Burn,
     )
 }
+
+pub(crate) fn utxos_from_genesis(genesis: &Genesis) -> TestBlockUtxos {
```

Comment:
Using `pub` is heavily discouraged. Usually we depend on the hierarchical design of the crate to limit the requirements of what's exposed. Make this `pub` and that's good enough.

Reply:
OK, thanks.